schemas/telemetry/debug-schema.json

```diff
@@ -35,4 +36,4 @@
       }
     }
   ]
-}
+}
```

Comment:
I think this missing eol is what makes prettier unhappy

Reply:
@xgouchet apparently, it was not enough 😅

Running the format command should help:
https://github.com/DataDog/rum-events-format/blob/473ab466ef0e0fbd854147b8335dc9b1c9ee152e/README.md?plain=1#L48-L50